ark/spLDA/processing.py

```diff
@@ -184,3 +187,121 @@ def create_difference_matrices(cell_table, features, training=True, inference=Tr
 
     matrix_dict = {"train_diff_mat": train_diff_mat, "inference_diff_mat": inference_diff_mat}
     return matrix_dict
+
+
+def gap_stat(features, k, clust_inertia, num_boots):
```

Comment:
`num_boots` should be given a default value.  I'm not sure what that value would be, but what ever you think is appropriate.